contracts/test/integration/post-proposal-checks/IntegrationTestRoles.sol

```diff
@@ -65,36 +65,26 @@ contract IntegrationTestRoles is PostProposalCheck {
             addresses.mainnet("GOVERNOR")
         );
 
-        // LIQUID_PCV_DEPOSIT_ROLE
+        // PCV_DEPOSIT_ROLE
+        assertEq(core.getRoleAdmin(VoltRoles.PCV_DEPOSIT), VoltRoles.GOVERNOR);
+        assertEq(core.getRoleMemberCount(VoltRoles.PCV_DEPOSIT), 4);
         assertEq(
-            core.getRoleAdmin(VoltRoles.LIQUID_PCV_DEPOSIT),
-            VoltRoles.GOVERNOR
-        );
-        assertEq(core.getRoleMemberCount(VoltRoles.LIQUID_PCV_DEPOSIT), 4);
-        assertEq(
-            core.getRoleMember(VoltRoles.LIQUID_PCV_DEPOSIT, 0),
+            core.getRoleMember(VoltRoles.PCV_DEPOSIT, 0),
             addresses.mainnet("PSM_DAI")
         );
         assertEq(
-            core.getRoleMember(VoltRoles.LIQUID_PCV_DEPOSIT, 1),
+            core.getRoleMember(VoltRoles.PCV_DEPOSIT, 1),
```

Comment:
Having the PSM as a PCV Deposit means funds can get moved to the PSM through the router instead of the allocator, thus bypassing the global rate limits. The PSM will also never call in to the oracle to update balances, so this should be removed.